tests/gtest/avif_fuzztest_helpers.cc

```diff
@@ -69,6 +69,41 @@ ImagePtr CreateAvifImage16b(size_t width, size_t height, int depth,
                          reinterpret_cast<const uint8_t*>(samples.data()));
 }
 
+std::vector<ImagePtr> CreateAvifAnim8b(size_t num_frames, size_t width,
+                                       size_t height,
+                                       avifPixelFormat pixel_format,
+                                       bool has_alpha,
+                                       const std::vector<uint8_t>& samples) {
+  std::vector<ImagePtr> frames;
```

Comment:
Maybe encapsulate the function in template in the .cc (with depth as a parameter and called here with `/*depth*/=8`)? Just in case the function becomes more complex? Not necessary,